data/json/professions.json

```diff
@@ -6265,5 +6305,605 @@
+    "points": 4,
+    "skills": [
+      { "level": 4, "name": "gun" },
+      { "level": 4, "name": "smg" },
+      { "level": 4, "name": "pistol" },
+      { "level": 3, "name": "speech" }
+    ],
+    "items": {
+      "both": {
+        "items": [ "socks", "suit", "wristwatch", "tie_skinny", "dress_shoes", "balclava", "chestrig", "gloves_medical", "kevlar" ],
+        "entries": [
+          { "item": "uzi", "ammo-item": "9mmfmj", "charges": 32, "custom-flags": [ "auto_wield" ] },
+          { "item": "uzimag", "ammo-item": "9mmfmj", "charges": 32 },
+          { "item": "uzimag", "ammo-item": "9mmfmj", "charges": 32 },
+          { "item": "uzimag", "ammo-item": "9mmfmj", "charges": 32 },
+          { "item": "duffelbag", "contents-group": "bank_score" },
+          { "group": "charged_two_way_radio" }
+        ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "bra", "boy_shorts" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "lookout",
+    "name": "Lookout",
+    "description": "You were tasked with scoping out the bank and staying on the lookout for any heat.  The frantic screams coming from your walkie-talkie were your clue to abandon ship.  You may have let your crew down, but you got away with your life.",
+    "points": 3,
+    "skills": [ { "level": 3, "name": "gun" }, { "level": 3, "name": "pistol" } ],
+    "items": {
+      "both": {
+        "items": [ "socks", "suit", "wristwatch", "tie_skinny", "dress_shoes", "binoculars", "kevlar" ],
+        "entries": [
+          { "item": "sw_619", "ammo-item": "357mag_fmj", "charges": 7, "container-item": "holster" },
+          { "item": "357mag_fmj", "charges": 20 },
+          { "group": "charged_two_way_radio" }
+        ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "bra", "boy_shorts" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "getaway",
+    "name": "Getaway Driver",
+    "description": "You were supposed to drive your accomplices to safety, but you decided to bail after bystanders started banging on your SUV.  You may have let your crew down, but you got away with your life.",
+    "points": 4,
+    "skills": [ { "level": 6, "name": "driving" }, { "level": 3, "name": "gun" }, { "level": 3, "name": "pistol" } ],
+    "vehicle": "suv",
+    "items": {
+      "both": {
+        "items": [ "suit", "gloves_leather", "socks", "dress_shoes", "wristwatch", "tie_skinny", "fancy_sunglasses", "kevlar" ],
+        "entries": [
+          { "group": "charged_two_way_radio" },
+          { "item": "sw_619", "ammo-item": "357mag_fmj", "charges": 7, "container-item": "holster" },
+          { "item": "357mag_fmj", "charges": 20 }
+        ]
+      },
+      "male": [ "briefs" ],
+      "female": [ "bra", "panties" ]
+    }
+  },
+  {
+    "type": "profession",
+    "id": "golfer",
+    "name": "Golfer",
```

Comment:
Again, hobby not profession

Reply:
there's such a thing as professional golfers

Reply:
> "description": "You were a golf enthusiast, playing with your friends on weekends.  Let's hope you practiced your swing.",